juju/juju
# [JUJU-1105] new_color_scheme_for_juju_status

## Please provide the following details to expedite review (and delete this heading)

*Replace with a description about why this change is needed, along with a description of what changed.*

## Checklist


## QA steps

```sh
juju status --color
```
*Check that the colors match the colors in this doc*
**Note** the section headers colors vary based on your default terminal's color
Juju [color doc](https://docs.google.com/document/d/1F8BhZmIG4w_SJfztIexiBOEikXmSz9d3Ju1zoqkNdbQ/edit)
